### ISSUE TITLE
ISA: Example in `isabelle/examples/AtomicBakeryG.thy` fails.

### ISSUE THREAD

**Issue by kape1395** [tlaplus/tlapm#133]
This example fails after upgrading to Isabelle2024.
It might be because of the Nat/Int rewrite, but I'm unsure.

**Comment by kape1395** [comment]
The last commit this example was working is eed59fc6db0f99b14a1860b9ddf2580eaebf3415,
It passes there with `Isabelle2024`,  and a minor fix in the ROOT file
```
$ git diff
diff --git a/isabelle/examples/ROOT b/isabelle/examples/ROOT
index cbcd3fa..8033e83 100644
--- a/isabelle/examples/ROOT
+++ b/isabelle/examples/ROOT
@@ -6,7 +6,7 @@ Examples of using Isabelle/TLA+
 *)
 
 (* build the session (without a heap) using "isabelle build -D ." *)
-chapter "TLA+ Examples"
+chapter "TLA+Examples"
 session "TLA-Examples" = "TLA+" +
   options [document=pdf, document_output = "output", document_variants="document:outline=/proof"]
```
Tried to build it as follows in the `isabelle/examples` folder
```
~/Programs/Isabelle2024/bin/isabelle build -d ../ -D . -f -c
```

The next commit touching `/isabelle/` is d44f1a1c64844a5bce80af73b1717a85c1593bdd; the example fails here.

@muenchnerkindl, maybe that's useful for you.

**Comment by kape1395** [comment]
Resolved in #109 and #134 .
